int32 id
float64 value

# enum: DEPTH, HEIGHT
int16 mode

float64 timeout
